tests/components/todoist/conftest.py

```diff
@@ -0,0 +1,61 @@
+"""Test fixtures for the Todoist integration."""
```

Comment:
I just moved these out of `test_calendar.py` so they could also be used in `test_coordinator.py`.  There are no changes to any of these fixtures.